lib/Jsrt/JsrtExternalObject.cpp

```diff
@@ -18,11 +18,12 @@ JsrtExternalType::JsrtExternalType(Js::ScriptContext* scriptContext, JsFinalizeC
         true)
         , jsFinalizeCallback(finalizeCallback)
 {
+    this->flags |= TypeFlagMask_JsrtExternal;
```

Comment:
I don't think this is right; the flag is for cross-context marshalling behavior where we don't add cross-site thunk on externalobject, and we force marshalling when externalobject is on the prototype chain. we don't do marshalling in JsrtExternalObject's object methods.